packages/api/cms-api/src/generator/generate-crud.ts

```diff
@@ -669,6 +715,18 @@ function generateRelationsFieldResolver({ generatorOptions, metadata }: { genera
 function generateResolver({ generatorOptions, metadata }: { generatorOptions: CrudGeneratorOptions; metadata: EntityMetadata<any> }): string {
     const { classNameSingular, fileNameSingular, instanceNameSingular, classNamePlural, fileNamePlural, instanceNamePlural } =
         buildNameVariants(metadata);
+
+    const {
+        resolverDecorator: resolverPermissionDecorator,
+        listDecorator: listPermissionDecorator,
+        singleDecorator: singlePermissionDecorator,
+        createDecorator: createPermissionDecorator,
+        updateDecorator: updatePermissionDecorator,
+        deleteDecorator: deletePermissionDecorator,
+    } = generateRequiredPermissionDecorators({
+        generatorOptions,
+        metadata,
+    });
```

Comment:
renaming here to make usage further down more readable. and it doesn't make sense to export it this way as the function already contains permission-decorator...